Multisig/Logic/Models/EIP712Transformer.swift

```diff
@@ -7,18 +7,26 @@
 //
 
 import Foundation
+import Version
 
 class EIP712Transformer {
     static func typedDataString(from transaction: Transaction) -> String {
         guard let safe = transaction.safe else { preconditionFailure("safe is required for typedData") }
 
-        let safeDomain = EIP712Domain(
+        var safeDomain = EIP712Domain(
             verifyingContract: safe.description
         )
+        var eip712Description = EIP712Domain.eip712Description
+
+        // Reference: https://github.com/gnosis/gnosis-py/blob/772a7da3a281c21931b8e5f01508a74cab7ecfb8/gnosis/safe/safe_tx.py#L169
+        if Version(transaction.safeVersion!)! >= Version("1.3.0")! {
```

Comment:
when can the `Version(transaction.safeVersion!)! ` crash?

Reply:
will make it safer